test/test_image.py

```diff
@@ -158,6 +159,45 @@ def test_encode_png(self):
                 RuntimeError, "The number of channels should be 1 or 3, got: 5"):
             encode_png(torch.empty((5, 100, 100), dtype=torch.uint8))
 
+    def test_read_1_bit_png(self):
+        try:
+            from cv2 import imread
+        except ImportError:
+            return
```

Comment:
If imread can't be imported the test will be marked as passed, when in reality it didn't run: maybe a `pytest.mark.skipif` mark would be better here.

Reply:
Can we replace the cv2.imread call with a PIL read? There is virtually no other dependency of cv2 anywhere else on the library.

Reply:
PIL read returns boolean values with 1-bit depth.